sdk/src/main/scala/com/horizen/account/state/GasPool.scala

```diff
@@ -17,15 +19,19 @@ class GasPool(initialGas: BigInteger) {
     if (currentGas.compareTo(gas) < 0) {
       throw new OutOfGasException()
     }
+    log.debug(s"subtracting $gas from currentGas=$currentGas")
```

Comment:
don't forget to remove the debug calls from this place. Or at least keep it as `trace` logs.
otherwise they will bring a lot of noise to us.

Reply:
Ok, done